### DISCUSSION TITLE
es/datasets/detect/open-images-v7/

### DISCUSSION THREAD

**Discussion by giscus** [ultralytics/ultralytics#16297] [Docs]
# es/datasets/detect/open-images-v7/

Explora el completo conjunto de datos Open Images V7 de Google. Infórmate sobre sus anotaciones, aplicaciones y uso YOLOv8 modelos preentrenados para tareas de visión por ordenador.

https://docs.ultralytics.com/es/datasets/detect/open-images-v7/

**Comment by MiguelTucto** [comment]
Hello! It's my first time training with yolov8. I need help about some processes. I've downloaded 3126 images from openimagesv7. These images are between fruits and vegetables. I trained with 50 epochs, but it stopped in 19 because It didn't present positive changes. After that, I've used the best.pt model exported to detect. When I use that model, It don't recognize how I expected the fruits and vegetable that I've trained. I need add more epochs? or maybe the quantity of fruits and vegetable have to be similar? Thank you in advance!
![image](https://github.com/user-attachments/assets/97ce5ad6-de07-42d5-9c44-0b0f3349e5f9)
![image](https://github.com/user-attachments/assets/b456569f-54e9-4e9b-a1e7-13e5eaa36a15)

**Reply by glenn-jocher** [reply]
@MiguelTucto it sounds like your model might benefit from more training data or balanced class representation. Consider increasing the number of images for each class and possibly extending the training epochs. You can also experiment with data augmentation to enhance model performance. For more detailed guidance, you might want to explore the Open Images V7 documentation.

**Reply by MiguelTucto** [reply]
Thanks dude! but I was thinking in two possibilities. The first one is to apply data augmentation and all images are between 2000 images. The second one is reduce all of images to 500 images. Which one will work better?

**Reply by glenn-jocher** [reply]
Applying data augmentation to your 2000 images would likely yield better results than reducing the dataset to 500 images. Augmentation can enhance model robustness by introducing variability, which is beneficial for training.

**Comment by UltralyticsAssistant** [comment]
👋 Hola @giscus[bot], ¡gracias por iniciar esta discusión sobre los conjuntos de datos Open Images V7 de Google! 🚀

Para aquellos interesados, recomendamos visitar la [documentación](https://docs.ultralytics.com/es/datasets/detect/open-images-v7/) donde se puede profundizar en sus anotaciones, aplicaciones y utilizar modelos preentrenados de YOLOv8 para tareas de visión por computadora.

Si este es un informe de 🐛 bug, por favor proporciona un [ejemplo mínimo reproducible](https://docs.ultralytics.com/help/minimum_reproducible_example/) para que podamos ayudarte mejor.

Si tienes una pregunta ❓ sobre entrenamiento personalizado, incluye todos los detalles posibles como ejemplos de imágenes del conjunto de datos y registros de entrenamiento. Asegúrate también de seguir nuestros [Consejos para los mejores resultados de entrenamiento](https://docs.ultralytics.com/guides/model-training-tips/).

Únete a la comunidad de Ultralytics donde más te convenga. Para chat en tiempo real, visita [Discord](https://ultralytics.com/discord) 🎧. ¿Prefieres discusiones a fondo? Echa un vistazo a [Discourse](https://community.ultralytics.com) o sumérgete en hilos de nuestro [Subreddit](https://reddit.com/r/ultralytics).

## Actualización

Actualiza al último paquete `ultralytics`, incluyendo todos los [requisitos](https://github.com/ultralytics/ultralytics/blob/main/pyproject.toml) en un entorno de [**Python>=3.8**](https://www.python.org/) con [**PyTorch>=1.8**](https://pytorch.org/get-started/locally/) para verificar que tu problema no esté ya resuelto en la última versión:

```bash
pip install -U ultralytics
```

## Entornos

YOLOv8 se puede ejecutar en los siguientes entornos verificados actualizados (con todas las dependencias incluyendo [CUDA](https://developer.nvidia.com/cuda)/[CUDNN](https://developer.nvidia.com/cudnn), [Python](https://www.python.org/) y [PyTorch](https://pytorch.org/) preinstalados):

- **Notebooks** con GPU gratuita: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/ultralytics/yolov8"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Máquina Virtual de Deep Learning. Consulta la [Guía rápida de GCP](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
- **Amazon** AMI de Deep Learning. Consulta la [Guía rápida de AWS](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
- **Imagen de Docker**. Consulta la [Guía rápida de Docker](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

## Estado

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml/badge.svg" alt="Ultralytics CI"></a>

Si esta insignia está en verde, todas las pruebas de [CI de Ultralytics](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml?query=event%3Aschedule) están pasando actualmente. Las pruebas de CI verifican el correcto funcionamiento de todos los [modos de YOLOv8](https://docs.ultralytics.com/modes/) y [tareas](https://docs.ultralytics.com/tasks/) en macOS, Windows y Ubuntu cada 24 horas y en cada commit.

Este es un mensaje automatizado y un ingeniero de Ultralytics también ayudará pronto. 😊
